app/_kong_plugins/json-threat-protection/index.md

```diff
@@ -135,7 +135,7 @@ rows:
 
 
 ## Log JSON request body violations
-In tap mode, if the plugin detects violations in the JSON request body, it logs a warning and proxies the request to the upstream service instead of blocking the request. In other words, in tap mode, the plugin only monitors the traffic.
+In tap mode, if the plugin detects violations in the JSON request body, it logs a warning and proxies the request to the Upstream Service instead of blocking the request. In other words, in tap mode, the plugin only monitors the traffic.
```

Comment:
```suggestion
In tap mode, if the plugin detects violations in the JSON request body, it logs a warning and proxies the request to the upstream service instead of blocking the request. In other words, in tap mode, the plugin only monitors the traffic.
```

Reply:
shouldn't this be lowercase if it's not a gateway service/upstream entity?